packages/tests/esm-misc/src/entities/codegen/AuthorCodegen.ts

```diff
@@ -204,13 +204,13 @@ export abstract class AuthorCodegen extends BaseEntity<EntityManager, string> im
   }
 
   get books(): Collection<Author, Book> {
-    return this.__data.relations.books ??= hasMany(
+    return (this.__data.relations.books ??= hasMany(
```

Comment:
You might already be on this, but can we setup `.prettierignore`/etcs for any of the codegen files? Just since it's ~expected the dprint formatter we use is "prettier-ish" and so will always have some drift...

Reply:
That's a shout. On codegen, I've just added #1231 if you're open to something like that.

Reply:
each test package has different prettierignores, so I've raised it to the root and ensured that all **/entities/codegen/* files are ignored.